pkg/driver/params.go

```diff
@@ -102,10 +102,8 @@ func NewVolumeParams(m map[string]string) (*VolumeParams, error) {
 // NewSnapshotParams parses the input params and instantiates new SnapshotParams.
 func NewSnapshotParams(m map[string]string) (*SnapshotParams, error) {
 	var err error
-	params := &SnapshotParams{ // set up defaults, if any.
-		SnapSize:    100,
-		AbsSnapSize: false,
-	}
+	params := &SnapshotParams{}
```

Comment:
In case of thick provisioning if `snapSize` is not present on the snapshotclass this will return snapSize as 0. right?

Reply:
Makes sense will be resolved in other if else condition in the caller functions